src/main/java/it/gov/pagopa/pu/workflow/wf/ingestionflow/debtposition/wfingestion/DebtPositionIngestionFlowWFImpl.java

```diff
@@ -103,4 +112,13 @@ private InstallmentIngestionFlowFileResult processFile(Long ingestionFlowFileId)
     }
     return ingestionResult;
   }
+
+  private String mergeErrorDescriptions(String ingestionResultErrorDescription, String additionalError) {
+    if(StringUtils.isEmpty(additionalError)){
+      return ingestionResultErrorDescription;
+    } else {
+      return (ingestionResultErrorDescription == null ? "" : ingestionResultErrorDescription)
+        + "\n\nThere were errors during the synchronization of the ingested Debt Position:" + additionalError;
```

Comment:
If not errorDescription, let's start the description with "There were errors..."
If there is a errorDescription, let's set an empty line to separate both sections
```suggestion
      return (ingestionResultErrorDescription == null ? "" : ingestionResultErrorDescription + "\n\n")
        + "There were errors during the synchronization of the ingested Debt Position:" + additionalError;
```